test/plots/linePlotTests.ts

```diff
@@ -1,412 +1,299 @@
 ///<reference path="../testReference.ts" />
 
 describe("Plots", () => {
-  // HACKHACK #1798: beforeEach being used below
   describe("LinePlot", () => {
-    it("entities() with NaN in data", () => {
-      let svg = TestMethods.generateSVG(500, 500);
-      let dataWithNaN = [
-        { foo: 0.0, bar: 0.0 },
-        { foo: 0.2, bar: 0.2 },
-        { foo: 0.4, bar: NaN },
-        { foo: 0.6, bar: 0.6 },
-        { foo: 0.8, bar: 0.8 }
-      ];
+    describe("Basic Usage", () => {
+      let svg: d3.Selection<void>;
+      let xScale: Plottable.Scales.Linear;
+      let yScale: Plottable.Scales.Linear;
+      let linePlot: Plottable.Plots.Line<number>;
+      let data = [{x: 0, y: 0}, {x: 1, y: 1}];
+      let dataset: Plottable.Dataset;
+
+      beforeEach(() => {
+        svg = TestMethods.generateSVG();
+        xScale = new Plottable.Scales.Linear();
+        yScale = new Plottable.Scales.Linear();
+        linePlot = new Plottable.Plots.Line<number>();
+        linePlot.x((d: any) => d.x, xScale);
+        linePlot.y((d: any) => d.y, yScale);
+        linePlot.attr("stroke", (d: any, i: number, m: any) => d3.rgb(d.x, d.y, i).toString());
```

Comment:
It's kind of odd that this gets set here, when it's only used in a few tests. Can we set it only in the tests where it's used?